common/src/main/java/com/viaversion/viabackwards/api/rewriters/LegacyBlockItemRewriter.java

```diff
@@ -185,6 +189,23 @@ public void register() {
         return item;
     }
 
+    @Override
+    public @Nullable Item handleItemToServer(@Nullable final Item item) {
```

Comment:
|customName also needs to be handled

Reply:
Already handled by the ItemRewriterBase class